packages/components/storybook/webpack.config.js

```diff
@@ -13,12 +13,12 @@ module.exports = {
         },
       },
       {
-        test: /\.(png|jpe?g|gif)$/,
+        test: /\.png|jpe?g|gif$/,
```

Comment:
@tbergq it's the same issue here and in other RegEx's as @mrtnzlml called out.

Reply:
🚀